### DISCUSSION TITLE
Overwrite t() to record missing translations

### DISCUSSION THREAD

**Discussion by robov** [intlify/vue-i18n#1263] [Q&A]
I would like to overwrite the t function in my app.js to record missing translations.
In the past (vue2) i created a `Vue.prototype.$t = function (...args) {}` that did the checking and finally called the i18n.t() to return some value
However I have no clue how to do this in vue3

The most I am struggeling how do I catch a call like
`
import { useI18n } from "vue-i18n";
const { t } = useI18n();
t('hello')
`
and redirect it to a function function and return the result of the translation
`
mytranslate(...args) {
return i18n.t(args)
}
`
When this is working I can record the missing translations etc.
Any suggestions are highly appreciated

**Comment by robov** [comment]
partial answer...
I can do this to catch the call from within my main application
```
        VueApp
            .use(plugin)
            .use(i18n);

        i18n.global.t = customTranslate; 
```

But when another npm package does:
```
import { useI18n } from "vue-i18n";
const { t } = useI18n();
t('hello);
```

Then the original is called and not the customTranslate

**Reply by noah-nuebling** [reply]
I believe the replacement for `Vue.prototype.$t` is `app.config.globalProperties.$t`. If you override that it should have the same effect (I haven't tested this and am no expert)
